src/coffee/mapping-utils/csv.coffee

```diff
@@ -38,16 +46,26 @@ class CsvMapping
     rows
 
   _getValue: (order, mapping) ->
+    attributeRegexp = /(lineItems\[.*]\.variant\.attributes)\.(.+)$/i
+    attributeNames = attributeRegexp.exec(mapping[0])
+
+    if (attributeNames isnt null)
+      attributes = access order, attributeNames[1]
+      return @formatProductAttributes(attributeNames[2], attributes)
+
     value = access order, mapping[0]
     if not value and value != 0
       return ''
 
     if _.size(mapping) is 2 and _.isFunction mapping[1]
-      mapping[1].call undefined, value
+      mapping[1].call this, value, mapping[0]
     else if isMoneyFormat value
       formatMoney value
     else
-      value
+      if mapping[0].match(/^lineItems\[.*]\.variant\.attributes\.?$/gi)
```

Comment:
could we add a comment here? I think it would help with reading the regex

Reply:
Commented here 867eadc